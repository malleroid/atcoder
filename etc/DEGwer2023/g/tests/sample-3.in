8
0.837078 0.492956 1
0.360579 -0.565500 0
-0.367448 -0.492394 1
0.491637 -0.658814 1
-0.505114 -0.538563 1
0.544637 0.592884 1
-0.622207 -0.379934 1
0.402129 0.684158 1
