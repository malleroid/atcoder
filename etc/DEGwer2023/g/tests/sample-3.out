28.879053537910
